src/app/3d/qgspointcloud3dsymbolwidget.cpp

```diff
@@ -569,3 +570,22 @@ void QgsPointCloud3DSymbolWidget::maximumScreenErrorChanged( double maxScreenErr
   mMaximumScreenError = maxScreenError;
   emitChangedSignal();
 }
+
+void QgsPointCloud3DSymbolWidget::showBoundingBoxesChanged( int checkBoxState )
+{
+  Q_UNUSED( checkBoxState );
+  bool showBoundingBoxes = mShowBoundingBoxesCheckBox->isChecked();
+  if ( showBoundingBoxes == mShowBoundingBoxes )
+    return;
+  mShowBoundingBoxes = showBoundingBoxes;
+  emitChangedSignal();
+}
+
+void QgsPointCloud3DSymbolWidget::setShowBoundingBoxes( bool showBoundingBoxes )
+{
+  if ( showBoundingBoxes == mShowBoundingBoxes )
+    return;
+  mShowBoundingBoxesCheckBox->setChecked( showBoundingBoxes );
+  mShowBoundingBoxes = showBoundingBoxes;
+  emitChangedSignal();
```

Comment:
We should not emit changed signal here - that one should be emitted only when user modifies something - but this is called for initialization of the widget...